packages/docs/src/styles/_sections.scss

```diff
@@ -54,7 +61,7 @@
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  height: $pt-grid-size * 13;
+  height: $pt-grid-size * 50;
```

Comment:
Feels weird that we have to explicitly set a height. Why not just have docs containers expand and contract automatically with the height of the child contents?